src/app/layouts/global/Header.tsx

```diff
@@ -1,133 +1,55 @@
-import { changeLanguage } from "i18next";
-import { Link } from "react-router-dom";
+import { useState } from "react";
+import { Logo, useTypedTranslation, BlueClose, Catalog, Present, Basket, Wishlist, UserAccount, Search } from "@shared";
+import { LanguageToggle, RoleLink } from "@src/components";
 
-import { useAppDispatch, useAppSelector } from "@store";
-import { ADMIN, GUEST_USER, MANAGER, useTypedNavigate, useTypedTranslation } from "@shared";
-import { authUser, setRole } from "@src/app/store/slices/user";
-import type { RoleUnion } from "@src/shared/types/User";
-// import { useLoginMutation } from "@src/app/api/auth";
-
-function LogButtons (): JSX.Element {
-  const navigate = useTypedNavigate();
-  const dispatch = useAppDispatch();
-  // const [login] = useLoginMutation();
-
-  const loginHandler = (event: MouseEvent<HTMLButtonElement>): void => {
-    // get role to check role action
-    const role = prompt("enter your role", "");
-
-    if (role === MANAGER) {
-      dispatch(setRole({
-        role: MANAGER,
-      }));
-    } else if (role === ADMIN) {
-      dispatch(setRole({
-        role: ADMIN,
-      }));
-    }
-
-    dispatch(authUser({
-      isAuth: true,
-    }));
-
-    navigate("/");
-
-    // try {
-    //   void login({
-    //     password: "",
-    //     username: "",
-    //   }).unwrap();
-    // } catch (error) {
-    //   console.log({
-    //     error,
-    //   });
-    // }
-  };
-
-  const logoutHandler = (event: MouseEvent<HTMLButtonElement>): void => {
-    dispatch(authUser({
-      isAuth: false,
-    }));
-    // set role to default when logout
-    dispatch(setRole({
-      role: GUEST_USER,
-    }));
-    // navigate("/");
-    // try {
-    //   void login({
-    //     password: "",
-    //     username: "",
-    //   }).unwrap();
-    // } catch (error) {
-    //   console.log({
-    //     error,
-    //   });
-    // }
-  };
-  return (
-    <div>
-      <button type="button" onClick={loginHandler}>Login</button>
-      <button type="button" onClick={logoutHandler}>Logout</button>
-    </div>
-  );
-}
-
-function LanguageToggle (): JSX.Element {
+export default function Header (): JSX.Element {
   const t = useTypedTranslation();
-
-  const handleChangeLanguage = (event: MouseEvent<HTMLButtonElement>): void => {
-    const { name } = event.target as HTMLButtonElement;
-    void changeLanguage(name);
-  };
+  const [search, setSearch] = useState("");
 
   return (
-    <>
-      {(["en", "uk"] as TranslationKeys[]).map((language, index) => (
-        <button
-          key={language}
-          name={language}
-          onClick={handleChangeLanguage}
+    <header>
+      <div className="flex justify-between relative z-10 bg-white px-[5%] items-center">
+        <Logo/>
+        <RoleLink/>
+        <LanguageToggle/>
+      </div>
+      <svg className="relative z-0 -top-[0.5rem]" width="100%" viewBox="0 0 20 0.7" style={{ filter: "drop-shadow(0px 4px 20px rgba(50, 64, 161, 0.5))" }}>
```

Comment:
Please use one of approachs - % or rem. Better is rem as tailwind use them under the hood. Just play with rem and find what u want.